Update README.md

kayoslab opened the pull request:
added caffe:ne to the Readme file

kayoslab committed:
Update README.md

added caffe:ne to the Readme file

Schemetrical commented:
Can you use "HealthKit & CoreAnimation", remove the -, and remove the Submitted, thanks!

kayoslab committed:
Update README.md

Changes @Schemetrical 'Can you use "HealthKit & CoreAnimation", remove the -, and remove the Submitted, thanks!'

Schemetrical commented:
Good luck! One more day!

kayoslab commented:
Ditto ;)
